src/components/BotBuilder/BotBuilder.js

```diff
@@ -297,6 +326,33 @@ class BotBuilder extends React.Component {
             this.setState({ openSnackbar: false });
           }}
         />
+        <Dialog
+          actions={[
+            <FlatButton
+              label="Cancel"
+              onClick={this.cancelDeleteAlert}
+              key={'Cancel'}
+              primary={true}
+              style={{ marginRight: '10px' }}
+            />,
+            <FlatButton
+              label="Delete"
+              onClick={this.closeDeleteAlert}
+              key={'Delete'}
+              backgroundColor={'#f50057'}
+              labelStyle={{ color: '#fff' }}
+              hoverColor={'#ff1744'}
+            />,
+          ]}
+          modal={false}
+          open={this.state.deleteAlert !== null}
+          onRequestClose={this.cancelDeleteAlert}
+        >
+          {this.state.deleteAlert === null ||
```

Comment:
`null` check not required

Reply:
null check is required otherwise it gives error (Cannot read property 'type' of null), the dialog component is mounted when the BotBuilder component mounts and not when its opened.